src/runtime-rs/crates/runtimes/src/manager.rs

```diff
@@ -87,10 +92,23 @@ impl RuntimeHandlerManagerInner {
             _ => return Err(anyhow!("Unsupported runtime: {}", &config.runtime.name)),
         };
         let runtime_instance = runtime_handler
-            .new_instance(&self.id, self.msg_sender.clone(), config)
+            .new_instance(&self.id, self.msg_sender.clone(), config.clone())
             .await
             .context("new runtime instance")?;
 
+        // initilize the trace subscriber
+        if config.runtime.enable_tracing {
+            let mut tracer = self.kata_tracer.lock().await;
+            if let Err(e) = tracer.trace_setup(
+                &self.id,
+                &config.runtime.jaeger_endpoint,
+                &config.runtime.jaeger_user,
```

Comment:
I'm wondering if a jaeger username with a jaeger password is required? Is it required by the crate or just we made the rule?

Reply:
`jaeger_user` and `jaeger_password` are optional. They are set for the authentication of the exporter.  I think that, sometimes, the upper collector needs this kind of information to identify us(the exporter). Thus we should better provide the configuration here.